controller/Controller.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * Controller logic for viewing pages and using the site.
+ *
+ * @author Keller Flint
+ */
+
+class Controller
+{
+    private $_f3; //router
+    private $_val;
+
+    /**
+     * Controller constructor
+     * @param $f3 Object The fat free instance
+     */
+    function __construct($f3)
+    {
+        //$this->_val = new Validation();
+        $this->_f3 = $f3;
+    }
+
+    /**
+     * Logic and rendering for tutors page
+     * @param string $year The year to load tutors data for
+     * @author Keller Flint
+     */
+    function tutorsPage($year)
+    {
+        global $db;
```

Comment:
Maybe we should pass in the db variable and create a private db variable  instead of calling global each time. This is more about the style.

Reply:
agreed